application.go

```diff
@@ -190,14 +182,14 @@ func (a *Application) Stop() {
 func (a *Application) Suspend(f func()) bool {
 	a.Lock()
 
-	if a.suspended || a.screen == nil {
+	if a.screen == nil {
 		// Application is already suspended.
```

Comment:
This comment is now not correct anymore. It should probably read `Screen has not yet been initialized.`.